monitors/onlinemon/include/SimpleStandardEvent.hh

```diff
@@ -11,7 +11,7 @@ typedef char int8_t;
 #include <string>
 #include <vector>
 #include <iostream>
-#include <stdint.h>
+//#include <stdint.h>
```

Comment:
It removes the nine errors in the second screen shot and it seems it is not necessary as it compiles without including stdint.h. The include files that appear in the error message are (indirectly) included by stdint.h.